scripts/deploy:push

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 set -e
-set -x
```

Comment:
I think deleting this from this script is the right move, but I think there are a few others (`scripts/build`) that benefit from `set -x`. Thoughts? I can't remember if you were summarily against it or not.